### ISSUE TITLE
Internet Explorer 11 Demo - [visibility] property does not work as expected

### ISSUE THREAD

**Issue by dck5132** [bertrandg/angular-split#192]
**Describe the bug**

When I toggle the visible property to false in the demo page <https://bertrandg.github.io/angular-split/#/examples/toggling-dom-and-visibility> in IE11 I see the gutter disappear as expected, but the actual split area is still visible

Screenshot Below:

![image](https://user-images.githubusercontent.com/18180081/64643236-f00b0d00-d3dd-11e9-87fe-3beb20f640bd.png)

This only occurs in IE 11, I have tried in the latest versions of Firefox and Chrome and it works as expected there.

Screenshot Below: 

![image](https://user-images.githubusercontent.com/18180081/64643366-36f90280-d3de-11e9-8597-686dce199601.png)

**To Reproduce**

Go to <https://bertrandg.github.io/angular-split/#/examples/toggling-dom-and-visibility> and toggle the visible property for one or more of the split areas in IE 11 browser

**Comment by vlodko** [comment]
@dck5132 this is the root cause [(link to file)](https://github.com/bertrandg/angular-split/blob/954682bc0a528df2a7c672dbf13039612dee7258/projects/angular-split/src/lib/component/split.component.scss#L36)
:
 ```
        &.as-hidden {
            flex: 0 1 0 !important;
            ...
        }
```

Since Internet Explorer 10 and 11 consider a unitless value in the flex-basis part to be syntactically invalid (and will thus be ignored), you need to include a unit in the flex-basis part of the flex shorthand value explicitly - `flex: 0 1 0% !important;`.

Overriding this value in my stylesheet resolved the issue.

@bertrandg , could you please add this change to your next release?

**Comment by beeman** [comment]
@dck5132 Today I merged #190 which should improve support for IE11. Can you verify if it works with `angular-split@4.0.0-beta.0`?
